radio/src/telemetry/frsky.cpp

```diff
@@ -48,7 +48,7 @@ enum FrSkyDataState {
 #endif
 };
 
-NOINLINE void processFrskyTelemetryData(uint8_t data)
+NOINLINE void processSerialData(uint8_t data)
```

Comment:
Sure I can make the change although strictly speaking the driver should be usage agnostic and as such should be called something like "serial_driver" and the process function "processSerialData". Looking at the history I saw several changes going back and forth with function and variable naming based on specific usages being handled at that specific time.